qa/tasks/vstart_runner.py

```diff
@@ -519,15 +519,14 @@ def list_connections():
         else:
             self._fuse_conn = new_conns[0]
 
-    def _run_python(self, pyscript):
+    def _run_python(self, pyscript, use_python3=False):
```

Comment:
could check `sys.version` instead.

Reply:
@tchaikov Wouldn't that check version of the interpreter test code is running on? `pysrcipt` would run on a separate interpreter (which should be python3, since `ceph_volume_client.py` due to previous commit is only py3 compatible).